script/upgrades/MU03/MU03Checks.sol

```diff
@@ -401,110 +395,82 @@ contract MU03Checks is Script, Test {
     console2.log("\tBreakerBox address updated in SortedOracles 🗳️");
   }
 
-  function verifyMedianDeltaBreaker() internal view {
+  function verifyMedianDeltaBreaker(MU03Config.MU03 memory config) internal view {
     // verify that cooldown period, rate change threshold and smoothing factor were set correctly
-    for (uint256 i = 0; i < poolConfigs.length; i++) {
-      if (poolConfigs[i].isMedianDeltaBreakerEnabled) {
-        uint256 cooldown = MedianDeltaBreaker(medianDeltaBreaker).rateFeedCooldownTime(
-          poolConfigs[i].referenceRateFeedID
-        );
-        uint256 rateChangeThreshold = MedianDeltaBreaker(medianDeltaBreaker).rateChangeThreshold(
-          poolConfigs[i].referenceRateFeedID
-        );
-        uint256 smoothingFactor = MedianDeltaBreaker(medianDeltaBreaker).smoothingFactors(
-          poolConfigs[i].referenceRateFeedID
-        );
+    for (uint256 i = 0; i < config.rateFeeds.length; i++) {
+      Config.RateFeed memory rateFeed = config.rateFeeds[i];
+
+      if (rateFeed.medianDeltaBreaker0.enabled) {
+        uint256 cooldown = MedianDeltaBreaker(medianDeltaBreaker).rateFeedCooldownTime(rateFeed.rateFeedID);
+        uint256 rateChangeThreshold = MedianDeltaBreaker(medianDeltaBreaker).rateChangeThreshold(rateFeed.rateFeedID);
+        uint256 smoothingFactor = MedianDeltaBreaker(medianDeltaBreaker).smoothingFactors(rateFeed.rateFeedID);
 
         // verify cooldown period
-        verifyCooldownTime(
-          cooldown,
-          poolConfigs[i].medianDeltaBreakerCooldown,
-          poolConfigs[i].referenceRateFeedID,
-          false
-        );
+        verifyCooldownTime(cooldown, rateFeed.medianDeltaBreaker0.cooldown, rateFeed.rateFeedID, false);
 
         // verify rate change threshold
         verifyRateChangeTheshold(
           rateChangeThreshold,
-          poolConfigs[i].medianDeltaBreakerThreshold.unwrap(),
-          poolConfigs[i].referenceRateFeedID,
+          rateFeed.medianDeltaBreaker0.threshold.unwrap(),
+          rateFeed.rateFeedID,
           false
         );
 
         // verify smoothing factor
-        if (smoothingFactor != poolConfigs[i].smoothingFactor) {
-          console2.log(
-            "MedianDeltaBreaker smoothing factor not set correctly for rate feed %s",
-            poolConfigs[i].referenceRateFeedID
-          );
-          revert("MedianDeltaBreaker smoothing factor not set correctly for all rate feeds");
+        if (smoothingFactor != rateFeed.medianDeltaBreaker0.smoothingFactor) {
+          console2.log("MedianDeltaBreaker smoothing factor not set correctly for cUSD/USDC %s", rateFeed.rateFeedID);
```

Comment:
Same here, we're not setting the breakers on `cUSD/USDC`. We're setting the breakers on `USDC/USD` oracles, that value is then used to power the `cUSD/USDC` pools, but the breakers don't act on or care about the `cUSD/USDC` pool's existence or rate.